terraform/environments/core-vpc/vpc.tf

```diff
@@ -263,6 +263,7 @@ resource "aws_iam_role" "member-delegation" {
           "Principal" : {
             "AWS" : concat(
               local.expanded_account_numbers_with_keys[each.key],
+              format("%s:role/github-actions", local.expanded_account_numbers_with_keys[each.key]),
```

Comment:
Actually thinking about this, if we've already allowed any of the accounts to assume the role with the statement on line 265, I don't think we need this do we?

Reply:
It's probably superfluous. 265 will expand out to something like `arn:aws:iam::111111111111:root` which ought to allow any user or role to assume the `member-delegation-$business_unit-$env` role.